config_deploy

```diff
@@ -0,0 +1,17 @@
+port 8080;
```

Comment:
Probably dont need this file on github twice, but also not important at this point.